docs/cloud/visualize/dashboards.md

```diff
@@ -9,22 +9,22 @@ Click on the **Dashboards** tab in any War Room to get started.
 
 From the Dashboards tab, click on the **+** button.
 
-![Add or manage
-dashboards](https://user-images.githubusercontent.com/1153921/108529360-a2145d00-7291-11eb-814b-2ea3303beb64.png)
+<img width="98" alt=" Green plus button " src="https://github.com/netdata/netdata/assets/73346910/511e2b38-e751-4a88-bc7d-bcd49764b7f6"/>
+
 
 In the modal, give your new dashboard a name, and click **+ Add**.
 
-Click the **Add Chart** button to add your first chart card. From the dropdown, select either *All Nodes** or a specific
+- The **Add Chart** button on the top right of the interface adds your first chart card. From the dropdown, select either **All Nodes** or a specific
```

Comment:
with the new charts even if you select a single node it is a composite chart - we removed the single instance charts

in this case it will have just one node selected 
![image](https://github.com/netdata/netdata/assets/82235632/1b3623f3-3d42-4613-bf19-f100b6569110)